specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,724 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-03-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Sql/locations/{locationName}/managedDatabaseRestoreAzureAsyncOperation/{operationId}/completeRestore": {
+      "post": {
+        "tags": [
+          "ManagedDatabases"
+        ],
+        "description": "Completes the restore operation on a managed database.",
+        "operationId": "ManagedDatabases_CompleteRestore",
+        "parameters": [
+          {
+            "name": "locationName",
+            "in": "path",
+            "description": "The name of the region where the resource is located.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "description": "Management operation id that this request tries to complete.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The definition for completing the restore of this managed database.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CompleteDatabaseRestoreDefinition"
```

Comment:
If regular database gets this API, will the resource type be the same? If no, then it should be something like CompleteManagedDatabaseRestoreDefinition